arrows-es6/index.html

```diff
@@ -0,0 +1,85 @@
+---
+feature_name: ES6 Arrow functions
+chrome_version: 45
+feature_id: 5047308127305728
+additional_head_content: <meta name="optional" value="optional">
+---
+
+<h3>Background</h3>
+
+<p>ES6 fat arrow functions have a shorter syntax compared to
+function expressions and lexically bind the <code>this</code> value. Arrow functions are always anonymous and effectively turn
+<code>function (arguments) { expression }</code> into arguments => expression. If using an expression after an arrow, the return is implicit, so no <code>return</code> is required.</p>
+
+{% capture initial_output_content %}
```

Comment:
You can remove this `{% capture initial_output_content %}` block. (It's just needed if you want something initial to show up in the output area.)